src/utils/axios.ts

```diff
@@ -2,12 +2,14 @@ import axios from "axios";
 
 // ----------------------------------------------------------------------
 
-const baseURL = "/api";
+const baseURL = "http://localhost:3000/api";
 
 const token = "testToken";
 
 axios.defaults.baseURL = baseURL;
 
 axios.defaults.headers.common = { Authorization: `bearer ${token}` };
```

Comment:
i dont you need "bearer", just the token. Test it without the word 'bearer', if it works, remove it.

Reply:
I haven't checked as there's no data. Also, currently, I tried to make token pass globally, but it needs useContext and it will wrap everything into client component. So if u make axios, u may have to put token inside the fetcher instead. Right now, we may place it like that because it's used for sign up